app/service/biz-service.js

```diff
@@ -88,5 +88,30 @@ function bizService($q, $log, $http, authService){
     });
   };
 
+  service.getPics = function(bizId) {
+    $log.debug('picService.getPics()');
+
+    return authService.getToken()
```

Comment:
In the context of a user editing/viewing their bizs, I can maybe see wrapping the call in getToken, but I would probably do it the same as I did for the findBizs() method. Definitely keep it for any method that alters/creates the data.